### PR TITLE
Clean up distribution

### PR DESCRIPTION
**Pull request by narthur157**
Wouldn't make this PR now, except that I need it for my discover-ractive branch on network to build. Was including too many things in the dist, resulting in es6 being uglified on network and bad times all around.

### CODE REVIEW

File: samples/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
   this is the container and it should be empty when ractive loads
   </div>
   <script src="../bower_components/requirejs/require.js" type="text/javascript"></script>
-  <script src="../config.js" type="text/javascript"></script>
+  <script src="/samples/config.js" type="text/javascript"></script>
```
Comment (Aintaer) [inline samples/index.html]: This is not safe at all. Should be `./config.js`

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ractive-partials",
   "scripts": {
-    "build": "babel --modules amd ./ractive-partials.js > dist/ractive-partials.js && echo built ractive-partials.js to dist/ractive-partials.js"
+    "build": "babel --modules amd ./src/ractive-partials.js > dist/ractive-partials.js && echo built ractive-partials.js to dist/ractive-partials.js"
```
Comment (Aintaer) [inline package.json]: Why the echo?

File: bower.json [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
-    "text": "~2.0.14",
     "module": "~1.0.4"
   },
   "ignore": [
-    "/samples/"
+    "/samples/",
+    ".*",
+    "*.json",
+    "*.md",
+    "ractive-partials.js",
```
Comment (Aintaer) [inline bower.json]: This is not necessary if moved to `dist/`

File: bower.json [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
+    "config.js"
```
Comment (Aintaer) [inline bower.json]: not necessary if moved to `samples/`

File: bower.json [hunk 5]
```diff
@@ -1,15 +1,19 @@
...
   "main": "ractive-partials.js",
   "dependencies": {
     "requirejs": "~2.1.18",
     "ractive": "~0.7.3",
-    "text": "~2.0.14",
     "module": "~1.0.4"
   },
   "ignore": [
-    "/samples/"
+    "/samples/",
```
Comment (Aintaer) [inline bower.json]: Why does this begin with `/`?

Comment (narthur157) [inline bower.json]: Because samples is in the root directory

Comment (narthur157) [inline bower.json]: I'll change it to '/', should be the same. I've seen this used in a lot of gitignores which is why I did it. Not sure why that tends to be a thing

File: package.json (hunk 2 above)
Comment (narthur157) [inline package.json]: Let's you know what it did, though I'd say it's pretty clear from the command being run in the terminal so I'll drop it

**Comment by mrjoelkemp** [comment]
:+1:

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ractive-partials",
   "scripts": {
-    "build": "babel --modules amd ./ractive-partials.js > dist/ractive-partials.js && echo built ractive-partials.js to dist/ractive-partials.js"
+    "build": "babel --modules amd ./src/ractive-partials.js > dist/ractive-partials.js"
```
Comment (Aintaer) [inline package.json]: use `babel inputfile -o outputfile` rather than the pipe redirect

**Comment by Aintaer** [comment]
:+1:

**Pull request merged**